app/models/comment.rb

```diff
@@ -1,2 +1,20 @@
+# == Schema Information
+#
+# Table name: comments
+#
+#  id         :bigint           not null, primary key
+#  text       :text
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  author_id  :bigint           not null
+#  post_id    :bigint           not null
+#
 class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+  belongs_to :post, counter_cache: true
+
+  #  updates the comments counter for a post
+  def update_comments_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
**OPTIONAL**

- [ ] It will be nice to use the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new comment has been saved to the database.

- [ ] Also make the `update_comments_counter` method private.